projects/schematics/README.md

```diff
@@ -15,19 +15,27 @@ Run the following command from your project root:
 - `useMetaTags`: Whether or not to configure baseUrl and mediaUrl in the meta tags from `index.html`
 - `featureLevel`: Application feature level. (default: '1.1')
 - `overwriteAppComponent`: Overwrite content of app.component.html file. (default: true)
+- `pwa`: Include PWA features while constructing application.
+- `ssr`: Include Server-side Rendering configuration.
 
 ### Other commands:
 By defaut `ng add @spartacus/schematics` will add all modules and files from commands listed below, but there will be situations where you want to run single command (i.e. for existing project).
```

Comment:
We could consider rewriting this line a little bit, as it does not fit as perfectly as before IMO.